tests/selenium/.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
```

Comment:
Exclude this file as it's only used from the root of a repository, and Treeherder already has one that runs the appropriate linting and tests.